.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ default_stages: [commit, push]
 default_language_version:
   python: python3
   node: 22.2.0
-minimum_pre_commit_version: '3.2.0'
```

Comment:
Is it possible to revert that quote change ? I guess it's vscode applying it automatically ?

Reply:
BTW. Likely we should apply some lint rule here by pre-commit - not sure if yamllint can be configured to expect `"`